src/main/groovy/jp/co/soramitsu/devops/docker/DockerPlugin.groovy

```diff
@@ -293,9 +295,11 @@ class DockerPlugin implements Plugin<Project> {
             return 'openjdk:13'
         } else if (javaVersion == 17) {
             return 'openjdk:17-jdk-slim'
+        } else if (javaVersion == 21) {
+            return 'amazoncorretto:21-alpine3.18-jdk'
```

Comment:
why not remain on the same provider as older versions? e.g. `openjdk:21-jdk-slim`

Reply:
The corretto images are way better maintained. 0 known vulnerabilities while the openjdk has quite a few issues. 

<img width="1276" alt="Screenshot 2024-04-03 at 11 30 07" src="https://github.com/soramitsu/gradle-sora-plugin/assets/10465468/3474c1af-a0e5-4feb-9775-58054d3f96d7">
<img width="1270" alt="Screenshot 2024-04-03 at 11 30 35" src="https://github.com/soramitsu/gradle-sora-plugin/assets/10465468/4923fecc-9875-41f5-a77e-ee9436adb327">
<img width="1269" alt="Screenshot 2024-04-03 at 11 37 07" src="https://github.com/soramitsu/gradle-sora-plugin/assets/10465468/09180a81-dbcd-4c8e-9d22-cbbbf1dbeb7e">

Reply:
I realize that, true, though those are default backup images and they're still widely used. In any corporate project a maintainer is intended to follow DevOps recommendations and use other suitable image

Reply:
This [image](https://hub.docker.com/_/openjdk) is officially deprecated and all users are recommended to find and use suitable replacements ASAP.
eclipse-temurin:21-jre image tested.